dv/uvm/icache/doc/ibex_icache_dv_plan.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "IBEX_ICACHE DV Plan"
+---
+
+<!-- Copy this file to hw/ip/ibex_icache/doc/ibex_icache_dv_plan.md and make changes as needed.
+For convenience 'ibex_icache' in the document can be searched and replaced easily with the
+desired IP (with case sensitivity!). Also, use the testbench block diagram
+located at OpenTitan team drive / 'design verification'
+as a starting point and modify it to reflect your ibex_icache testbench and save it
+to hw/ip/ibex_icache/doc/tb.svg. It should get linked and rendered under the block
+diagram section below. Please update / modify / remove sections below as
+applicable. Once done, remove this comment before making a PR. -->
```

Comment:
Since you've presumably done the find/replace, this comment needs deleting! (See last sentence :-))

Reply:
Done